frontend/src/views/CategoryPage.vue

```diff
@@ -1,7 +1,5 @@
 <template>
-  <v-content>
-    <ItemsDisplay />
-  </v-content>
+  <ItemsDisplay />
```

Comment:
As far as I know, for proper rendering, you have to wrap the component in a `v-content` as per vuetify guidelines. Any reason as to why you have changed this?

Reply:
I did that to remove that extra white space before contents are displayed.

Reply:
Okay I see